docs/evaluate/major-components.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: major-components
+title: Temporal's major components
+description: todo
```

Comment:
Temporal's major components are a Temporal SDK, Temporal Service, Temporal UI, and Temporal CLI.